docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS/Kubernetes clusters
+description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 11/23/2018
+---
+# Build ASP.NET Core 2.1 applications deployed as Linux containers into AKS/Kubernetes orchestrator
+
+Azure Kubernetes Services (AKS) is Azure's managed Kubernetes orchestrations services that simplify container deployment and management.
+
+AKS main features are, an Azure-hosted control plane, automated upgrades, self-healing, user configurable scaling, and a simpler user experience for both developers and cluster operators.
+
+In the following examples we'll explore the creation of an ASP.NET Core 2.1 application, using Visual Studio 2017 that runs on Linux and deploys to an AKS Cluster in Azure.
+
+## Creating the ASP.NET Core 2.1 Project using Visual Studio 2017
+
+ASP.NET Core is a general-purpose development platform maintained by Microsoft and the .NET community on GitHub. It is cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and embedded/IoT scenarios.
+
+For this example, we use a simple project based on a Visual Studio Web API template, so you don't need any additional knowledge to create the sample, you only have to create the project using a standard template that includes all the elements to run a small project with a REST API, using ASP.NET Core 2.1 technology.
+
+![Add new project window in Visual Studio, selecting ASP.NET Core Web Application.](media/create-aspnet-core-application.png)
+
+**Figure 4-36**. Creating ASP.NET Core Application
+
+To create the sample project, you have to select **New > Project** on Visual Studio and ASP.NET Core Web Application
+
+Visual Studio will show a list with Templates for Web Projects. For our example you must select **Web API,** that means you will create an ASP.NET Web API Application.
+
+Verify that you have selected ASP.NET Core 2.1 as the framework. The .NET Core 2.1 is included in the last version of Visual Studio 2017 and is automatically installed and configured for you when you install Visual Studio 2017.
```

Comment:
you have -> you've 

as the framework -> as the target framework

The .NET Core -> .NET Core 

Visual Studio 2017 is a bit repetitive in the last sentence and you need to select the proper workload for that to happen. Maybe:
.NET Core 2.1 SDK is automatically installed and configured for you when you install Visual Studio 2017 with the .NET Core cross-platform development workload selected.

Also it needs to be version 15.7.0 or higher not any version of 2017